cviceni 09

adelapetrekova opened the pull request:
vytvoreni slozky cviceni 9

adelapetrekova committed:
vytvoření cviceni 09

misavojte commented:
conda config --add channels conda-forge

misavojte commented:
conda install weasyprint

misavojte commented:
```
timestamp = 0

html = """
<!DOCTYPE html>
<html>
<head>
    <title>Test</title>
    <link rel="stylesheet" type="text/css" href="style.css?{timestamp}">
</head>
<body>
    <h1>Test</h1>
    <p>This is a test.</p>
    <p>Timestamp: {timestamp}</p>
</body>
</html>
"""

with open("index.html", "w") as f:
    f.write(html.format(timestamp=timestamp))

print("Generated index.html")

# output html as a PDF
# first, import the necessary modules

from weasyprint import HTML

# then, convert the HTML to a PDF
HTML("index.html").write_pdf("index.pdf")

print("Generated index.pdf")
```

adelapetrekova committed:
vytvoření jupiter ntb se semestrálním projektem
rozdělení tvého posleného kodu do jednotlivých bunek a pridani komentaru k jednotlivým castem kodu pro lepsi orientaci a naslednou praci pri pridavani promennych atd

adelapetrekova committed:
vytvoření komentářů na konec jupiter co dále patří udělat a řešit

adelapetrekova committed:
vytvoreni bloku kodu kde si pisu co je potreba na pristi hodinu jako ukol a hruby nastrel co zhruba budu říkat
potreba dodelat informace o knihovne

adelapetrekova committed:
doplnení informací o knihovně

misavojte reviewed (commented):
Super, díky za přípravu:) Zatím nemerguji, ponechávám na další práci

adelapetrekova committed:
přidání dalších komentářů

adelapetrekova committed:
náčrt jak dát style do kodu a ne jako samostatny soubor

adelapetrekova committed:
odstraneni z hlavičky ten odkaz na css soubor
